src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,32 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder buildAppend = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder report = new StringBuilder();
```
краще називати змінні відповідно до їх призначення а ен типу данних